pmoves/services/consciousness-service/main.py

```
@@ -0,0 +1,226 @@
+    """
+    Generate CGP packet from consciousness theory.
+
+    Maps theory dimensions to geometric coordinates for Hi-RAG v2.
+    """
+    if not cgp_mapper:
+        raise HTTPException(status_code=503, detail="CGP mapper not initialized")
+
+    try:
+        theory_dict = theory.model_dump()
+        packet = cgp_mapper.theory_to_constellation(theory_dict)
+        return {"status": "success", "packet": packet}
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)